internal/config/example_data/config.yml

```diff
@@ -0,0 +1,32 @@
+clusterName: dummy_cluster
```

Comment:
I would call this folder as testdata instead of example_data, the Go tool will ignore any directory in your $GOPATH that starts with a period, an underscore, or matches the word testdata, as stated in https://dave.cheney.net/2016/05/10/test-fixtures-in-go

Reply:
🚀  Thanks! I did not know that!